polyply/data/martini3_beta/citations.bib

```diff
@@ -0,0 +1,19 @@
+@article{PEO_M2,
+  title={Transferable MARTINI model of poly (ethylene oxide)},
+  author={Grunewald, Fabian and Rossi, Giulia and de Vries, Alex H and Marrink, Siewert J and Monticelli, Luca},
+  journal={The Journal of Physical Chemistry B},
+  volume={122},
+  number={29},
+  pages={7436--7449},
+  year={2018},
+  publisher={ACS Publications},
+  doi={10.1021/acs.jpcb.8b04760}
+}
+
+@article{polyply,
+  title={Polyply; a python suite for facilitating simulations of (bio-) macromolecules and nanomaterials},
+  author={Grunewald, Fabian and Alessandri, Riccardo and Kroon, Peter C and Monticelli, Luca and Souza, Paulo CT and Marrink, Siewert J},
+  journal={Nature Communications},
```

Comment:
following https://www.nature.com/articles/s41467-021-27627-4, you can already add: 
```
volume={13},
pages={68},
```